<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slots::Internal::GeneralSettingsPage</class>
 <widget class="QWidget" name="Slots::Internal::GeneralSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="layersGroupBox">
     <property name="title">
      <string>Layers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QToolButton" name="removeLayerButton">
        <property name="text">
         <string>rem</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QTableView" name="layersTableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="addLayerButton">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sizeGroupBox">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="rowsLabel">
        <property name="text">
         <string>Rows:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="rowsSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="columnsLabel">
        <property name="text">
         <string>Columns:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="columnsSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
